src/Dialect/ONNX/ONNXOps/Math/ElementwiseBroadcast.cpp

```diff
@@ -28,10 +28,10 @@ using namespace onnx_mlir;
 namespace {
 
 // Verify shape for numpy style broadcasting operators.
-template <class OP, class ADAPTOR>
+template <class OP_TYPE>
```

Comment:
Had the same thought that `class ADAPTOR` was redundant when I was trying to use this function in NNPA.

Reply:
Yep, I saw this code from someone here, and decided to start using it more